docs/sources/guides/whats-new-in-v6-3.md

```diff
@@ -130,6 +130,8 @@ belonging to an LDAP group that gives them access to Grafana.
 
 Built-in support for SAML is now available in Grafana Enterprise.
 
+[See docs]({{< relref "auth/saml.md" >}})
```

Comment:
cannot find this file (it needs to be in OSS repo as well)

Reply:
done!